Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -1183,7 +1186,6 @@
 				54C9EDEE2040E16300A969CD /* Frameworks */,
 				54C9EDEF2040E16300A969CD /* Resources */,
 				EA424838F4A5DD7B337F57AB /* [CP] Embed Pods Frameworks */,
-				6BD54D799442CEB09349B73E /* [CP] Copy Pods Resources */,
```

Comment:
Yeah, it's a version thing. Since we're potentially all on random versions, every time we run pod update, cocoapods decides to rearrange the entire file. :(

FWIW, I just upgraded earlier today an am now on 1.5.3 (I think; not at my mac right now. Whatever the most recent version is.)